.github/workflows/odh-manifests-PR-sync.yml

```diff
@@ -0,0 +1,64 @@
+name: odh-manifests sync
+
+run-name: Sync manifests in odh-manifests
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  send-pull-requests:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout data-science-pipelines-operator repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: main
+          repository: opendatahub-io/data-science-pipelines-operator
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Send pull-request
+        run: |
+          LATEST_TAG=$(git describe --tags --always --abbrev=0)
+          REPOSITORY="opendatahub-io/odh-manifests"
+          FOLDER="bin/$REPOSITORY"
+          BRANCH_NAME="chore-update-scripts-to-$LATEST_TAG"
+
+          # Clone the remote repository and change working directory to the
+          # folder it was cloned to.
+          git clone \
+            --depth=1 \
+            --branch=master \
+            https://opendatahub-io:${{ secrets.ACCESS_TOKEN }}@github.com/$REPOSITORY \
+            $FOLDER
+          cd $FOLDER
+
+          # Setup the committers identity.
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git config user.name "GitHub Actions"
+
+          # Create a new feature branch for the changes.
+          git checkout -b $BRANCH_NAME
+          echo "Created branch: $BRANCH_NAME"
+
+          # Copy DSPO manifests. Using rsync to allow filtering of paths/files (e.g. like a .gitignore, hidden files, etc)
+          rsync -av --delete ../../../config/ data-science-pipelines-operator/
```

Comment:
we don't want _everything_ sync'd, just the manifests required to deploy dspo, so we want to **exclude**: 

* overlays
* samples 
* internal

Reply:
Done, updated the rsync to exclude these directories.